docs/coach.rst

```diff
@@ -48,7 +48,7 @@ Review Exercise Progress
 When a learner answers exercise questions in Kolibri, the progress bar below the exercise is taking into account only the most recent given answers, meaning that the learner must complete the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
 
 	.. figure:: img/get-5-correct.png
-	    :alt: If the exercise requires 5 correct check marks, learner must provide 5 correct answers one after another. Marks for each answer are located behind the button Check
+	    :alt: If the exercise requires 5 correct check marks, learner must provide 5 correct answers one after another. Marks for each answer are located behind the button :guilabel:`CHECK`
```

Comment:
This will be rendered as the alternative text for the image, and I don't see that using the `guilabel` is necessary (a part from not being able to render properly, right?)... :thinking:

Reply:
Oh right, haha, I wasn't reading that :)